<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QMainWindow" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1259</width>
    <height>707</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>CrossCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>530</y>
      <width>131</width>
      <height>124</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mainScreen">
     <item>
      <widget class="QPushButton" name="playButton">
       <property name="font">
        <font>
         <family>Terminal</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="creditsButton">
       <property name="font">
        <font>
         <family>Terminal</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Credits</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="howtoplayButton">
       <property name="font">
        <font>
         <family>Terminal</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>How to Play</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitButton">
       <property name="font">
        <font>
         <family>Terminal</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>230</y>
      <width>241</width>
      <height>150</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="multiplayerScreen">
     <item>
      <layout class="QHBoxLayout" name="playersLayout">
       <item>
        <layout class="QVBoxLayout" name="player1Layout">
         <item>
          <widget class="QLabel" name="player1Label">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <family>Terminal</family>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Player 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="player1Name">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <family>System</family>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhLatinOnly</set>
           </property>
           <property name="text">
            <string>Benedict</string>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="dragEnabled">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string notr="true"/>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="player2Layout">
         <item>
          <widget class="QLabel" name="player2Label">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <family>Terminal</family>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Player 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="player2Name">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <family>System</family>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhLatinOnly</set>
           </property>
           <property name="text">
            <string>Tomoe</string>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="dragEnabled">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string notr="true"/>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="seedLabel_2">
         <property name="font">
          <font>
           <family>Terminal</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Seed:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="seed_multi">
         <property name="font">
          <font>
           <family>System</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="playButton_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Terminal</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Play!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="gobackButton_5">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Terminal</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Go Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>230</y>
      <width>241</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="singleplayerScreen">
     <item>
      <widget class="QLabel" name="playerLabel">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Terminal</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Player</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="playerName">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>System</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhLatinOnly</set>
       </property>
       <property name="text">
        <string>Benedict</string>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string notr="true"/>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="seedLabel_1">
         <property name="font">
          <font>
           <family>Terminal</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>seed:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="seed_single">
         <property name="font">
          <font>
           <family>System</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="playButton_single">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Terminal</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Play!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="gobackButton_4">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Terminal</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Go Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>600</y>
      <width>87</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="howtoplayScreen">
     <item>
      <widget class="QLabel" name="directions">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Terminal</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>git gud</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="gobackButton_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Terminal</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Go Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>390</y>
      <width>153</width>
      <height>124</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="playScreen">
     <item>
      <widget class="QPushButton" name="singleplayerButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Terminal</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Single-Player</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="multiplayerButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Terminal</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Multiplayer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadgameButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Terminal</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Load Game</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="gobackButton_1">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Terminal</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Go Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_6">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>530</y>
      <width>90</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="creditsScreen">
     <item>
      <widget class="QLabel" name="credits">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Terminal</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>All: Me.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="gobackButton_3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Terminal</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Go Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1259</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
